README.md

```diff
@@ -111,26 +134,12 @@ Arangopipe represents metadata as a graph. The nodes of the graph above are the
 
 The data associated with the nodes and edges of the graph are stored as documents. The documents do not have a fixed structure and represent data using key-value pairs. This offers flexibility and permits users to decide the metadata elements that they would like to store. This also permits users to store metadata from any machine learning tech stack in Arangopipe.
 
-## Installing Arangopipe
-**Arangopipe** is available as a docker image. Docker containers for both _tensorflow_ and _torch_ are available. To run Arangopipe:
-
-1.  Start the ArangoML container. For _tensorflow_:
-
-    `docker run -it -p 6529:8529 -p 8888:8888 ap_tensor_flow:0.1`
-
-    for _torch_:
-    
-    `docker run -it -p 6529:8529 -p 8888:8888 ap_torch:0.1`
-
-2. Execute a `docker ps` command to get the ID of the running container. You can then get to a shell in the container using the `docker exec` command. Once you are in the container shell, you can generate test data to try **Arangopipe** using the `test_data_generator` utility provided with **Arangopipe**. The details are shown in the figure below.
-
-    <!--- ![Test Data Generation](test_data_generation.png) --->
-    <img src="test_data_generation.png" height="400">
-
-    You can then point your browser to `http://localhost:8888`. When prompted for a password, enter     `root` You will then see an examples notebook that explains how the examples are laid out in the directory. The examples illustrate how arangopipe can work with various tools in a machine learning stack. Examples of integration with _hyperopt_, _scikitlearn_, _tensorflow_, _torch_ and _mlflow_ are provided. 
+## Installing Arangopipe API
+To install the **Arangopipe** API, you would need to do the following:
 
-3. To use the **Arangopipe** user interface, point your browser to `http://localhost:3000`. To sign into the user interface, use a username of `root` and `open sesame` as password (note, there is a space between open and sesame). You neeed to complete the steps above to use the user interface. The features of the user interface are provided in the next section.
+1. `pip install python-arango == 4.4.0`
```

Comment:
Can't we preinstall that in the image?

Reply:
Actually this is pre-installed in the docker image. This is incase someone wants to use the API outside of the docker image. If you think that is confusing, I will just remove the whole install section.